### ISSUE TITLE
Dataloader shuffle In Test Set

### ISSUE THREAD

**Issue by MarioAvolio** [mrdbourke/pytorch-deep-learning#665]
Link: https://github.com/mrdbourke/pytorch-deep-learning/blob/main/05_pytorch_going_modular.md#2-create-datasets-and-dataloaders-data_setuppy

There is a problem, in the file "shuffle=True"
 ```
test_dataloader = DataLoader(
        test_data,
        batch_size=batch_size,
        **shuffle=False,  # don't usually need to shuffle testing data**
        num_workers=num_workers,
        pin_memory=True,
    )
```

**Comment by mrdbourke** [comment]
Thank you! 

Just fixed it :)
